components/alert/src/Alert.js

```diff
@@ -0,0 +1,128 @@
+/* @jsx jsx */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { jsx, useTheme, paint } from '@westpac/core';
+
+import { Button } from '../../button/src';
+import { CloseIcon } from '../../icon/src';
+
+import { CSSTransition } from 'react-transition-group';
+
+// ==============================
+// Component
+// ==============================
+
+export const Alert = ({ appearance, closable, icon: Icon, children }) => {
+	const [open, setOpen] = useState(true);
+	const { breakpoints, alert } = useTheme();
+	const mq = paint(breakpoints);
+
+	const styleCommon = {
+		marginBottom: alert.marginBottom,
+		padding: alert.padding,
+		paddingRight: [18, 30],
```

Comment:
Should use tokens

Reply:
Hey Jonny, my code is now looking like: `padding: getPaddingSM(),`
If I compare to yours, you have: `[alert.padding, getPaddingSM()]`
The added padding if closeable only applies to sm res and up?

Reply:
```padding: [alert.padding, getPaddingSM()]``` ensures for XS we use the default 18px padding, but for SM+ if there is a close button it will use 30px on the right; otherwise it will simply use the default 18px.

What you have will apply the 30px padding for all breakpoints if there's a close button. In XS we float the close button to the right, so the text wraps around the button. The button is no longer pos: abs, so we dont need additional padding.